schema/ingestion_config/migrations/issue_997.py

```diff
@@ -0,0 +1,192 @@
+"""
+This script migrates the ingest config files to a new schema based on the changes outlines in
+https://github.com/chanzuckerberg/cryoet-data-portal/issues/997
+"""
+import glob
+import itertools
+import json
+import traceback
+from typing import Union
+
+import numpy as np
+import yaml
+
+
+def rawtilts_to_collection_metadata(config: dict) -> None:
+    list_globs = []
+    if 'rawtilts' in config:
+        for i in config['rawtilts']:
+            if "sources" not in i:
+                continue
+            old_source = i["sources"][0]["source_multi_glob"]["list_globs"]
+            list_globs.extend(s for s in old_source if s.endswith('.mdoc') or "{mdoc_name}" in s)
+            for source in list_globs:
+                old_source.remove(source)
+        if list_globs:
+            if 'collection_metadata' not in config:
+                config['collection_metadata'] = [{"sources": [{"source_multi_glob": {"list_globs": []}}]}]
+            config["collection_metadata"][0]["sources"][0]["source_multi_glob"]["list_globs"].extend(list_globs)
+
+def rawtilts_to_alignments(data: dict) -> None:
```

Comment:
I believe this has been moved to `ingestion_tools/scripts/transform_ingestion_configs.py`. Do we still want this here?